letsencrypt/acme/other.py

```diff
@@ -0,0 +1,86 @@
+"""JSON objects in ACME protocol other than messages."""
```

Comment:
I am personally not a huge fan of the module name other.  Is there anything more specific we could name it?  I know I named a similar module in Apache obj.py (I don't think there would be name collisions in the modules).  Do you have any better ideas?